submissions/dummy_submission/the-nuclear-options-v1.0.md

```diff
@@ -0,0 +1,233 @@
+
+But what about the risks?
+
+ 
+
+## Nuclear Safety in Factorio: A Brief History
+
+Nucelar power was something that Wube wanted to get around to for quite a while and they shared some early design ideas before releasing the version know in 0.15. When they shared their first implementation tinkers in [FFF#164](https://www.factorio.com/blog/post/fff-164), there was a lot of focus on heating and cooling water and steam. The early designs involved high-temperature steam being cooled and only partially consumed in turbines, then condensed back into water at 95C in cooling towers, and then being fed back into heat exchangers. You can imagine this being difficult to balance, leading perhaps to one thing or another getting stuck pretty quickly. After a few more weeks of working on it, Wube decided around [FFF#167](https://www.factorio.com/blog/post/fff-167) that dealing with so much temperature management can be avoided and that water and steam could simply be treated as two separate fluids with fixed temperatures, hence simplifying regular boilers too. 
+
+Meanwhile, the effects of temperature on the reactor itself were also softened: At first it was hinted that a nuclear reactor that heated up uncontrollably would explode, hence reactors would demand constant monitoring as in real life. This idea was softened before release though, such that a nuclear plants uses its internal emergency cooling systems to never let it itself heat up beyond 1000C, effectively allowing you to safely leave it alone. Well, mostly. Beware that it still *can* explode if is at over 900C and gets destroyed by a reckless player or biter. And if other reactors are close by, this may lead to [quite a light show](https://www.youtube.com/watch?v=l4ePIuhbiPg&ab_channel=Trupen). Despite the trivialized safety, we can see that Wube has provided a nuclear system which could still get pretty interesting and risky via the added complexity from the neighbor bonus. For those who would have preferred the high-risk and more realistic approach, there are multiple [mods]() that offer a look into the problems of meltdowns and radiation hazards.
+
+ Now that we' seen how nuclear power is not so dangerous in the vanilla game, let's look over the other common concerns about it.
+
+## **Common Concerns with Nuclear Power**
+
+* **Nuclear power is too expensive to set up until the late game** – In terms of research, you do not need more than chemical science packs to unlock nuclear power technology. The setting up requires time, yes. And a single nuclear reactor sure is expensive. However the material costs, in terms of iron and copper, for a 40MW starter level nuclear setup is only [a fraction]()<!--[See appendix]--> of the cost of setting up 40MW worth of solar panels and accumulators, and the cost per MW of adding any further reactors is substantially cheaper thanks to already-established infrastructure and the substantial gains from the neighbor bonus. 

```

Comment:
you compare the cost of 40MW to solar. Can you also compare to the cost of a 40MW coal-steam engine power plant and a nuclear-fuel-steam engine power plant?

Reply:
Could be interesting to add a 40MW plant to, but a big cost then is the fuel you need to feed it, because it's very inefficient. So you'd have to add that to your comparison too.

I think it's also important to mention that you don't have to build a massive plant at the start. You can do a small easy one, just like you say you normally do. Starting with a small one will help you understand the mechanics.

Reply:
The "starting small" point should be highlighted for sure. I also will include Trez's point about nuclear taking up far less space than solar, but maybe another part of the article is more suitable for that, I'll see. 

I'll also include steam power to the cost comparison.  To reduce clutter, I am inclined to make the calculations into a reddit post and I'll put the conclusions to the article with a link to the post.

Reply:
you can make full calculations elsewhere, yes. but main calculus can definitely be placed here as illustrations of your content.